README.md

```diff
@@ -132,6 +135,7 @@ If you want to use the global:
 ```html
 <!-- Load dependencies -->
 <script src="https://unpkg.com/react/dist/react.js"></script>
+<script src="https://packd.now.sh/prop-types?name=PropTypes"></script>
```

Comment:
This is assuming the service works (which it's not right now) and it has no SLA. Not a huge deal because this is really only for demos and toys anyway...